Formula/metabase.rb

```diff
@@ -21,6 +21,8 @@ class Metabase < Formula
   def install
     if build.head?
       Language::Node.setup_npm_environment
+      # help yarn find npm's node-gyp binary inside superenv
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/lib/node_modules/npm/bin/node-gyp-bin"
```

Comment:
Is there any harm to simply chucking it into the `$PATH` unconditionally when `"language/node"` is `require`d?